Vagrantfile

```diff
@@ -107,4 +107,10 @@ SCRIPT
     # We want provision to be run with the permissions of the vagrant user.
     privileged: false,
     inline: $provision_script
+  
+  if File.file?("tools/custom_provision.sh")
+    config.vm.provision "shell",
+      privileged: false,
+      path: "tools/custom_provision.sh"
+  end
```

Comment:
I think it might be simpler to just move this if statement into the `SCRIPT` above; what do you think?